test/data/test_multi_embedding_tensor.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import torch
+
+from torch_frame.data.multi_embedding_tensor import MultiEmbeddingTensor
+
+
+def test_size():
+    num_rows = 8
+    num_cols = 3
+    lengths = torch.tensor([1, 2, 3])
+    offset = torch.tensor([0, 1, 2, 3])
+    values = torch.rand((num_rows, lengths.sum().item()))
+    met = MultiEmbeddingTensor(
+        num_rows=num_rows,
+        num_cols=num_cols,
+        values=values,
+        offset=offset,
+    )
+
+    assert met.size(0) == num_rows
+    assert met.size(1) == num_cols
+    with pytest.raises(ValueError, match="not have a fixed length"):
+        met.size(2)
+    with pytest.raises(IndexError, match="Dimension out of range"):
+        met.size(3)
+
+    assert met.shape[0] == num_rows
+    assert met.shape[1] == num_cols
+    assert met.shape[2] == -1
+
+
+def test_cat():
+    num_rows = 3
+    num_cols = 2
+    offset = torch.tensor([0, 2, 4])
+    values = torch.tensor([
```

Comment:
Use `rand` to generate tensor.

Reply:
I prefer this way to random tensors for readability, but I can replace this if you'd advocate for it